### PR TITLE
shortened headers

### LINKED ISSUE #70: Shorten headers
image

* they are really long
* make them short, play around with object fit classes and overflow classes
* you can also go in and make the image smaller if necessary

### PR DESCRIPTION
**Pull request by znobunaga**
- shortened headers to look like the figma file.
- text-resize is off for the board component. Causes the header images to shrink when making the screen smaller
- closes #70 

event_ai_2_screenshot
event_ai_screenshot
meet_board_2_screenshot
meet_board_screenshot

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -4,13 +4,17 @@ import wireframe from "/public/wireframe.webp";
 const Header = ({ text }) => {
   return (
     <div className="bg-ai-blue-400 flex items-center justify-between">
-      <Image className="" src={wireframe} alt="Left wireframe" />
+      <Image className="w-2/6 h-auto" src={wireframe} alt="Left wireframe" />
```
Comment (khandrew1) [inline src/components/Header.jsx]: try not to use h-auto, i want to shorten the heighten of the header cuz its really big right now. you might want to look into resizing the entire parent div.

**Comment by znobunaga** [comment]
- updated headers now making the headers shorter vertically
events_updated
events_updated2

**Comment by khandrew1** [comment]
could we make the text smaller just on mobile as well

**Comment by znobunaga** [comment]
- updated header text size to fit on mobile/smaller screens

header_textsize
header_textsize2

**Review by khandrew1 — APPROVED** [review]

**Pull request merged**